includes/parser/Parser.php

```diff
@@ -1264,6 +1264,11 @@ function doMagicLinks( $text ) {
 					[0-9Xx]                 # check digit
 					\b)
 			)!xu', array( &$this, 'magicLinkCallback' ), $text );
+
+		if ( preg_last_error() !== 0 ) {
+			\Wikia\Logger\WikiaLogger::instance()->error( 'PCRE error', [ 'preg_last_error' => preg_last_error() ] );
```

Comment:
Please add `"exception" => new Exception()` field to the context. This will allow us to have a full backtrace.